CHANGELOG.md

```diff
@@ -14,6 +14,9 @@
 - `Tree::tabs`: an iterator over the tabs in a tree. ([#53](https://github.com/Adanos020/egui_dock/pull/53))
 - `Style` now includes an option to show the hoverd tab's name. ([#56](https://github.com/Adanos020/egui_dock/pull/56))
 
+### Breaking changes
+- `Style` now includes an option to change default inner_margin ([#67](https://github.com/Adanos020/egui_dock/pull/67))
```

Comment:
Move this point to "Added", and here write about `TabViewer::inner_margin` being renamed to `TabViewer::inner_margin_override`.